tests/integration/tos.feature

```diff
@@ -3,7 +3,7 @@ Feature: Testing the Tos API
 
     Scenario: Create a ToS entry
         Given that I want to make a new "tos"
-        And that the request "Authorization" header is "testbasicuser"
+        And that the request "Authorization" header is "Bearer testbasicuser"
```

Comment:
@crcommons weirdly these were wrong from the beginning :/ I'm not really sure how these tests passed with an invalid auth header.
Going to test a little more before merge